src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/Verktoylinje.tsx

```diff
@@ -31,57 +34,99 @@ const OppgaveknapperStyle = styled.div`
     }
 `;
 
+const SvartLenkeKnapp = styled.button<{ aktiv?: boolean }>`
```

Comment:
Hoh, denne ble litt tung å lese. Men kan du kanskje bruke `<EkspanderKnapp/>` her? Eventuelt trekke denne ut i en egen fil?

Reply:
`<EkspanderKnapp/>` er blå, men hvis det er sort som gjelder her vil jeg anbefale å utvide den med en optional prop `sort` istedenfor å lage en ny og nesten lik knapp

Reply:
I like it.